src/services/print.js

```diff
@@ -374,6 +374,10 @@ ngeo.Print.prototype.encodeVectorStyle_ =
   } else if (!goog.isNull(imageStyle)) {
     this.encodeVectorStylePoint_(styleObject.symbolizers, imageStyle);
   }
+
+  if (!goog.isNull(textStyle) && textStyle.getText()) {
```

Comment:
Or maybe remove that test and test that within the `encodeTextStyle_` function. I think that would make more sense.